tests/src/main/java/com/hazelcast/simulator/tests/map/MapMaxSizeTest.java

```diff
@@ -163,9 +154,7 @@ public void timeStep(MapOperation operation) {
                     operationCounter.get++;
                     break;
                 case CHECK_SIZE:
```

Comment:
The default test configuration has a probability of 10% to execute the CHECK_SIZE operation, which will lead to an `UnsupportedOperationException` after this PR. Why did you delete all the code, instead of just setting the CHECK_SIZE prob to 0.0 (or remove it completely)?